f5lbaasdriver/v2/bigip/service_builder.py

```diff
@@ -203,6 +210,18 @@ def _get_extended_member(self, context, member):
 
         return (member_dict, subnet, network)
 
+    @log_helpers.log_method_call
+    def _get_extended_qos(self, context, loadbalancer):
+        try:
+            sess = keystone.get_session()
+            client_sess = client.Client(session=sess)
+            project = client_sess.projects.get(project=loadbalancer.tenant_id)
+            return project.qos
+
+        except Exception as e:
+            LOG.error('Exception: Get keystone project: %s', e.message)
```

Comment:
should we change the log level from error to warning or some lower levels?

Reply:
> should we change the log level from error to warning or some lower levels?

OK,  I will change this bit in this branch :)